src/coreclr/build-test.cmd

```diff
@@ -577,8 +581,10 @@ REM ============================================================================
 
 if NOT "%__LocalCoreFXPath%"=="" (
     echo Patch CoreFX from %__LocalCoreFXPath%
+    set __CoreFXBuildType=Debug
```

Comment:
This should be the opposite, assume it is release. If the build type is debug then use a debug corefx. That way we can correctly use a release corefx build with a checked runtime.

Reply:
Fixed, thanks for pointing that out. Ultimately we may want to introduce options for [semi-]arbitrarily combining configuration modes for the individual components (CoreCLR, CoreFX, tests in either subrepos - not sure what degrees of freedom we have) but I think it will be easier to address in a follow-up change.